repos/system_upgrade/el7toel8/actors/firewalldisable/actor.py

```diff
@@ -29,35 +14,35 @@ class FirewallDisable(Actor):
 
     def stop_firewalld(self):
         ''' Stops FirewallD '''
-        call(['systemctl', 'stop', 'firewalld'])
+        check_output(['systemctl', 'stop', 'firewalld'])
```

Comment:
I think that wherever you don't check the output, it doesn't makes sense to call `check_output`. Use `subprocess.call` instead.
It's 5 times in this file and one time in prepareupgradetransaction/actor.py.

Reply:
What you wrote is reason to use `check_output`, right? :) From help:
>  If the return code was non-zero it raises a CalledProcessError

Reply:
Well, it's nice that it raises CalledProcessError but actor catches the error. If this error is raised it would generate nice and shiny traceback for the user.
But this rant is not that relevant to this PR. We'll change the stdlib call in future to use the @shaded-enmity's https://github.com/oamg/leapp/pull/387 under the hood. Then, we'll need to update all actors again to accept the new return values.